applications/GeoMechanicsApplication/custom_utilities/process_factory.cpp

```diff
@@ -22,9 +22,7 @@ ProcessFactory::ProductType ProcessFactory::Create(const std::string& rProcessCl
                                                    const Parameters&  rProcessSettings) const
 {
     auto pos = mCreatorMap.find(rProcessClassName);
-    if (pos == mCreatorMap.end()) {
-        return nullptr;
-    }
+    KRATOS_ERROR_IF(pos == mCreatorMap.end()) << "Unexpected process (" << rProcessClassName << "), calculation is aborted";
```

Comment:
We have already discussed how we can make this more generic by using a callback mechanism.

Reply:
Done